### PR TITLE
feat(icons): add percentage icon

### PR DESCRIPTION
**Pull request by mguoynes**
## Summary

add percentage icon because this design calls for it https://www.figma.com/design/EGmc0e9SX1i5pk75ukLBMH/RG-refresh?node-id=1246-8127&node-type=instance&m=dev

## Screenshots (if appropriate):

Screenshot 2024-10-28 at 3 54 29 PM

### CODE REVIEW

File: packages/icons/src/img/sprite.svg [hunk 1]
```diff
@@ -451,25 +451,6 @@
         clip-rule="evenodd"
       />
     </symbol>
-    <symbol viewBox="0 0 20 20" id="lp-icon-chart-cube-root">
```
Comment (Niznikr) [inline packages/icons/src/img/sprite.svg:454]: We don't want to delete these

File: packages/icons/src/types.ts [hunk 2]
```diff
@@ -196,6 +196,7 @@ const icons = [
 	'package-outline-bolt',
 	'package-send',
 	'password',
+	'percentage',
```
Comment (Niznikr) [inline packages/icons/src/types.ts:199]: should be after `pause`

**Review by Niznikr — DISMISSED** [review]
Looks like you accidentally deleted a couple icons 😅

**Comment by mguoynes** [comment]
> Member

Fixed this

**Comment by mguoynes** [comment]
We decided to go another direction and this icon is not strictly needed.

**Comment by carmenquan** [comment]
@mguoynes or @mattdaviscodes (whoever is driving this now) In the thread Launchpad team mentions we need to change to the tabler % icon
https://launchdarkly.slack.com/archives/C03DH955TMW/p1729779071501719

**Review by Niznikr — APPROVED** [review]

**Pull request merged**